Plugins/Object/NWScript/nwnx_object.nss

```diff
@@ -206,7 +206,8 @@ void NWNX_Object_RemoveIconEffect(object obj, int nIcon);
 /// @brief Export an object to the UserDirectory/nwnx folder.
 /// @param sFileName The filename without extension, 16 or less characters.
 /// @param oObject The object to export. Valid object types: Creature, Item, Placeable, Waypoint, Door, Store, Trigger
-void NWNX_Object_Export(object oObject, string sFileName);
+/// @param sAlias The alias of the resource directory to add the .git file to. Default: UserDirectory/nwnx
+void NWNX_Object_Export(object oObject, string sFileName, string sAlias = "NWNX");
```

Comment:
Linux has case-sensitive filesystems by default. Are you sure that the default directory will be `nwnx`, not `NWNX`?

Reply:
yeap. all folders in nwn are always upcased